src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -323,6 +320,31 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
         }
 
         [OuterLoop] // TODO: Issue #11345
+        [ConditionalTheory(nameof(BackendSupportsCustomCertificateHandlingAndClientSupportsDHECipherSuites))]
+        [MemberData(nameof(CertificateValidationServersAndExpectedPolicies))]
+        public async Task UseCallback_BadCertificate_ExpectedPolicyErrors(string url, SslPolicyErrors expectedErrors)
+        {
+            if (PlatformDetection.IsUap)
+            {
+                // UAP HTTP stack caches connections per-process. This causes interference when these tests run in
+                // the same process as the other tests. Each test needs to be isolated to  own process.
+                // See dicussion: https://github.com/dotnet/corefx/issues/21945
+                RemoteInvoke((remoteUrl, remoteExpectedErrors) =>
```

Comment:
>In UAP test mode, does the RemoteInvoke-spawned process run under the context of the test runner's UAP app package (low IL, inside an AppContainer, etc)? In other words: are we sure that this will still test the UWP scenario?

@tarekgh Can you answer these questions?

I will say that for UAPAOT test runs, the .NET Native compiled app is NOT a Appx (Store App). Instead it is a native win32 app.  PlatformDetection.IsUap returns true for both UAP and UAPAOT test runs.  I think, though, that the test runner still uses app-container (low IL) but is not a packaged AppX.  @yizhang82 can you clarify that?

In this particular test case, all we need is process isolation and so, the test will pass as long as those test conditions exist.

But in the future, we want to add more tests and so we will need to make sure we get app-container / AppX isolation